src/config/useAuth.js

```diff
@@ -0,0 +1,5 @@
+export default function useAuth() {
+  const auth = true;
+
+  return auth;
+}
```

Comment:
this function always return true, is it for testing? If yes, please add //TODO comment to not forget to change it later